### ISSUE TITLE
Value of scale for iframes is unclear in the spec

### ISSUE THREAD

**Issue by alijuma** [bokand/ViewportAPI#52]
Chrome always returns scale 1 for iframes, and this is also covered by viewport-scale-iframe-manual.html. But it's not clear that the spec language at https://wicg.github.io/visual-viewport/index.html#dom-visualviewport-scale implies this. Does this follow from the math given there or do we need to add a special case?

This came up in a WebKit review (https://bugs.webkit.org/show_bug.cgi?id=179385#c7).

**Comment by bokand** [comment]
Yes, the intent is that an iframe's visual viewport doesn't have any scale of its own when the page is zoomed. You could imagine if we ever added per-iframe based zoom it would be non-1. The meaning of it is that it's "the scale _relative_ to the parent frame". I'll make the spec clearer on this point.
